README.md

```diff
@@ -67,9 +67,12 @@ Whenever you change dependencies (adding, removing, or updating, either in `pack
 * The `allow-scripts` configuration in `package.json`
   * Run `yarn allow-scripts auto` to update the `allow-scripts` configuration automatically. This config determines whether the package's install/postinstall scripts are allowed to run. Review each new package to determine whether the install script needs to run or not, testing if necessary.
   * Unfortunately, `yarn allow-scripts auto` will behave inconsistently on different platforms. macOS and Windows users may see extraneous changes relating to optional dependencies.
-* The LavaMoat auto-generated policy in `lavamoat/node/policy.json`
-  * Run `yarn lavamoat:auto` to re-generate this policy file. Review the changes to determine whether the access granted to each package seems appropriate.
+* The production LavaMoat policy files (`lavamoat/browserify/*/policy.json`)
+  * Run `yarn lavamoat:auto` to re-generate these policy files. Review the changes to determine whether the access granted to each package seems appropriate.
```

Comment:
The `lavamoat:auto` script will re-generate both the prod and build system policy files. Is this what you meant?

```suggestion
  * Run `yarn lavamoat:background:auto` to re-generate these policy files. Review the changes to determine whether the access granted to each package seems appropriate.
```